PCbuild/pcbuild.proj

```diff
@@ -18,7 +18,9 @@
 
   <ItemDefinitionGroup>
     <FreezeProjects>
-      <Platform>$(PreferredToolArchitecture)</Platform>
+      <Platform>$(Platform)</Platform>
+      <Platform Condition="$(Platform) == 'ARM'">Win32</Platform>
+      <Platform Condition="$(Platform) == 'ARM64'">x64</Platform>
```

Comment:
This does not look right to me.  The point is to build `_freeze_module.exe` using the same architecture as the build toolchain, so that the host can run `_freeze_module.exe`.  Making this change would prevent a native build on ARM64 from running `_freeze_module.exe`, because it would be built for x64 (unless there's some automagical emulation involved to allow an x64 exe to run on ARM64, but that sounds painful).